resources/ApprovePage.js

```diff
@@ -0,0 +1,22 @@
+const restClient = new mw.Rest();
+
+function sendApprovalRequest( approve ) {
+	const pageId = mw.config.get( 'wgArticleId' );
+	const endpoint = `/page-approvals/v0/page/${ pageId }/${ approve ? 'approve' : 'unapprove' }`;
+	const csrfToken = mw.user.tokens.values.csrfToken;
+
+	restClient.post( endpoint, { token: csrfToken } )
+		.then( response => {
+			console.log( 'Request successful:', response );
+			mw.notify( 'Request successful: ' + JSON.stringify( response ) );
```

Comment:
This is temporary, right?

Reply:
Yes. Since we don't have working BE endpoints.